Controller/Heroes/Necro/Cards/TalismanCardController.cs

```diff
@@ -30,7 +30,8 @@ public override void AddTriggers()
         {
             //That target is immune to damage from undead targets.
             base.AddImmuneToDamageTrigger((DealDamageAction dd) => this.IsUndead(dd.DamageSource.Card) && base.IsThisCardNextToCard(dd.Target));
-            base.AddTrigger<DestroyCardAction>(dca => IsThisCardNextToCard(dca.CardToDestroy.Card), dca => dca.GameController.MoveCard(dca.DecisionMaker, this.Card, this.Card.Owner.PlayArea, actionSource: dca, playCardIfMovingToPlayArea: false, doesNotEnterPlay: true), TriggerType.MoveCard, TriggerTiming.After, ignoreBattleZone: true);
+
+            base.AddIfTheCardThatThisCardIsNextToLeavesPlayMoveItToTheirPlayAreaTrigger(true,true);
```

Comment:
This change would have some of the same problems that Stranger's Rune's had.  commiting a change to make Talesmin and Runes behave the same.